app/src/main/java/ru/vssemikoz/newsfeed/MainActivity.java

```diff
@@ -16,36 +18,35 @@
 import retrofit2.Response;
 import retrofit2.Retrofit;
 import retrofit2.converter.gson.GsonConverterFactory;
+import ru.vssemikoz.newsfeed.DAO.NewsItemDAO;
+import ru.vssemikoz.newsfeed.DataBase.NewsAppDataBase;
 import ru.vssemikoz.newsfeed.adapters.NewsFeedAdapter;
 import ru.vssemikoz.newsfeed.api.NewsApi;
 import ru.vssemikoz.newsfeed.models.NewsApiResponseItem;
 import ru.vssemikoz.newsfeed.models.NewsApiResponse;
 import ru.vssemikoz.newsfeed.models.NewsItem;
 
 public class MainActivity extends AppCompatActivity {
-    String KEY = "c94a57cbbb50497f94a2bb167dc91fc5";
-    String MAIN_URL = "https://newsapi.org";
-
+    MainApplication globalVariables;
```

Comment:
`MainApplication` is main application, not global variables. Be extra careful choosing names for your variables, classes, etc. Once you found a name, return to it before commit and think about once again - you should review your code yourself before every single commit and before making a PR.